bedrock/firefox/templates/firefox/os/ver/2.5.html

```diff
@@ -0,0 +1,109 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/os/index.html" %}
+
+{% block body_class %}{{ super() }} v2-5{% endblock %}
+
+{% block extrahead %}
+  {% stylesheet 'firefox_os_new' %}
+{% endblock %}
+
+{% block hero_copy %}
+  <div class="hero-content">
+    <h1>{{ _('Experience Firefox OS on your Android device')}}</h1>
+    <p>{{_('Introducing Firefox OS 2.5 Developer Preview, an experimental app that lets you to use Firefox OS on your Android device.')}}</p>
+    <div class="primary-cta-signup signup">
+      <a href="#TODO" class="cta-button" role="button">{{_('Get the Android App')}}</a>
```

Comment:
Yeah that's the missing urls I still need. I'll add in the rel="external" since it's most likely the google play store.